crates/bevy_ui/src/update.rs

```diff
@@ -35,37 +35,47 @@ fn update_clipping(
     children_query: &Query<&Children>,
     node_query: &mut Query<(&Node, &GlobalTransform, &Style, Option<&mut CalculatedClip>)>,
     entity: Entity,
-    clip: Option<Rect>,
+    maybe_inherited_clip: Option<Rect>,
 ) {
-    let (node, global_transform, style, calculated_clip) = node_query.get_mut(entity).unwrap();
+    let (node, global_transform, style, maybe_calculated_clip) =
+        node_query.get_mut(entity).unwrap();
+
     // Update this node's CalculatedClip component
-    match (clip, calculated_clip) {
-        (None, None) => {}
+    match (maybe_inherited_clip, maybe_calculated_clip) {
```

Comment:
I was going to switch them because it's a good idea, I like semantic ordering etc, but then I realised that it's probably better to use nested `if let` statements as you avoid that which option is which ambiguity completely.